tests/unit/store/index.spec.js

```diff
@@ -14,7 +14,6 @@ describe('index', () => {
     describe('api', () => {
       it('contains multiple resources', () => {
         expect(Store.state.api.traction).toBeDefined()
-        expect(Store.state.api.printMyBarcode).toBeDefined()
```

Comment:
I think this should be tested still - if you remove the v2 from pmb api config name